contrib/intel/jenkins/Jenkinsfile

```diff
@@ -122,6 +122,21 @@ pipeline {
                      }
 
                  }
+		 stage('ze-shm') {
+		     agent {node {label 'eth'}}
+		     steps{
+                        withEnv(['PATH+EXTRA=/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin/:$PYTHONPATH'])
+                        {
+                          sh """
+                            env
+                            (
+                                cd  ${env.WORKSPACE}/contrib/intel/jenkins/
+                                python3.7 runtests.py --prov=shm --ofi_build_mode='dbg'
+                            )
+                          """
+                        }
+		     }
+		 }
```

Comment:
CI testing only, I know, but could you align this block with the other blocks are the same level

Reply:
yes